test/test_model.py

```diff
@@ -61,6 +61,29 @@ class TestModelFunctionality(unittest.TestCase):
     def setUp(self):
         pass
 
+    def __helper_single_letter_species_names(self,model,x):
+        try:
+            model.add_species(spatialpy.Species(x,0))
+            print("checking '"+x+"'")
+        except Exception as e:
+            raise e
+
+    def test_single_letter_species_names(self):
+        """ Test that we can have any single letter species name"""
+        model = spatialpy.Model()
+        reserved_names = ["x", "vol","sd","data_fn","t","debug_flag","Spatialpy"]
+        for x in string.ascii_letters:
```

Comment:
Recommend using subTest with for loop to provide more information if the test fails
`with self.subTest(name=x):`

this will eliminate the need for the print statement on line 67